components/dom_crawler.rst

```diff
@@ -73,8 +73,9 @@ tree.
 .. note::
 
     If you need better support for HTML5 contents or want to get rid of the
-    inconsistencies of PHP's DOM extension, install the `html5-php library`_.
-    The DomCrawler component uses it automatically when available.
+    inconsistencies of PHP's DOM extension, install the `html5-php library`_
+    and the DomCrawler component will use it automatically when the content has
```

Comment:
I don't agree with combining the sentences with an "and". I think shorter sentences are better.

Reply:
Done!